platform/android/java/lib/src/org/godotengine/godot/Godot.java

```diff
@@ -916,7 +917,9 @@ private boolean obbIsCorrupted(String f, String main_pack_md5) {
 	}
 
 	public boolean gotTouchEvent(final MotionEvent event) {
-
+		if ((event.getSource() & InputDevice.SOURCE_MOUSE) == InputDevice.SOURCE_MOUSE) {
+			return true;
+		}
```

Comment:
Override this method in `GodotEditor.java` and move this logic there:
```
@Override
public boolean gotTouchEvent(final MotionEvent event) {
    if ((event.getSource() & InputDevice.SOURCE_MOUSE) == InputDevice.SOURCE_MOUSE) {
        return true;
    }
    return super.gotTouchEvent(event);
}
```

Reply:
This is not editor specific implemention, the point for it is to give full mouse control to android